inc/zoo/swar/SWAR.h

```diff
@@ -0,0 +1,179 @@
+#pragma once
+
+/// \file Swar.h SWAR operations
+
+#include "metaLog.h"
```

Comment:
I think relative inclusion paths are counterproductive: I like the ability to have parts of the inclusion tree in different file paths and let the compilation merge them into a single one. 
Change to include `zoo/swar/metaLog.h` or perhaps move this to the top level as a top-level utility

Reply:
Also, I have this custom:
1. Include first the most specific headers, in this case, the ones in the `zoo` library.
2. For each section, leave a space.  In this case a blank line separating the inclusion of metaLog from type-traits